src/app/routes/utils/sharedDataTransformers/addIndexesToEmbeds/index.js

```diff
@@ -20,16 +20,26 @@ const getEmbedProvider = getOembedProp('provider_name');
 const getEmbedIndexOfType = getOembedProp('indexOfType');
 const matchesEmbedProvider = provider =>
   pipe(getEmbedProvider, equals(provider));
-const enrichBlocks = (accumulator, block) => {
+const accumulatedBlocksByProvider = (accumulator, block, index, blocks) => {
```

Comment:
I think you got mixed up with my suggestion the other day

```suggestion
const enrichBlocks = (accumulator, block, index, blocks) => {
```